app/models/hearing_summary.rb

```diff
@@ -19,6 +19,10 @@ def hearing_days
     body['hearingDays'].map { |hearing_day| hearing_day['sittingDay'] }
   end
 
+  def hearing_resulted?
```

Comment:
Minor nitpick: should we name this just `resulted?` to be able to use it as `hearing.resulted?`, rather than `hearing.hearing_resulted?`